search-app/test/cmr/search/test/site/routes.clj

```diff
@@ -88,13 +91,24 @@
       (is (string/includes?
            (:body response)
            "Documentation for Search"))
+      (is (string/includes?
+           (:body response)
+           "v dev"))
       (is (string/includes?
            (:body response)
            "site/docs/search/api"))
       (is (string/includes?
            (:body response)
            "site/docs/search/site")))))
 
+(deftest dynamic-release-version
+  (let [_ (side-api/eval-form `(common-config/set-release-version! "1.0"))
```

Comment:
You can move the call to `set-release-version` above the `let` block instead of doing it in a binding. 

Since there's another test that relies on the default version you should change the version back at the end of this test so that it doesn't matter what order tests run in.

e.g. add this at the end:
```
(side-api/eval-form `(common-config/set-release-version! "dev"))
```

Reply:
okay. I was just thinking of that but since the test didn't fail, I thought it was okay not to change it. Will do that. Thanks.